### PR TITLE
Release 1.1.14

### PR DESCRIPTION
**Pull request by mkniewallner**
## Fixed

- Fixed an issue where dependencies hashes could not be retrieved when locking due to a breaking change on PyPI JSON API ([#5973](https://github.com/python-poetry/poetry/pull/5973))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1177,7 +1183,8 @@ Initial release
 
 
 
-[Unreleased]: https://github.com/python-poetry/poetry/compare/1.1.13...1.1
+[Unreleased]: https://github.com/python-poetry/poetry/compare/1.1.14...1.1
+[1.1.13]: https://github.com/python-poetry/poetry/releases/tag/1.1.14
```
Comment (finswimmer) [inline CHANGELOG.md:1187]:
```suggestion
[1.1.14]: https://github.com/python-poetry/poetry/releases/tag/1.1.14
```

**Review by finswimmer — CHANGES_REQUESTED** [review]
Some minor things (see comments).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
 # Change Log
 
+## [1.1.14] - 2022-07-08
+
+## Fixed
+
+- Fixed an issue where dependencies could not be retrieved when locking them due to a breaking change on PyPI JSON API ([#5973](https://github.com/python-poetry/poetry/pull/5973))
```
[the remark this answers is not in the document]
Comment (mkniewallner) [reply CHANGELOG.md:7]: Yes you're right, bad phrasing, will rework that.

[the remark this answers is not in the document]
Comment (mkniewallner) [reply CHANGELOG.md]: Amended in https://github.com/python-poetry/poetry/pull/5974/commits/457fac209b4400ab2e2544b20b01f75501646098.

**Review by finswimmer — APPROVED** [review]
Thank you so much!

Just still waiting for the checks to complete .... :clock1:

**Pull request merged**